uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Relationship.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class CRM_HRCore_Test_Fabricator_Relationship {
+
+  private static $defaultParams = [
+    'sequential' => 1,
+    'name_a_b' => "test AB",
+    'name_b_a' => "test BA",
+    'contact_type_a' => "Individual",
+    'contact_type_b' => "Individual",
+  ];
+
+  public static function fabricateRelationshipType($params = []) {
```

Comment:
Relationship and RelationshipType are two different entities, so I think it's better to have two separate fabricators, unless you have a good reason to have this fabricateRelationshipType method in this fabricator